lib/inputs/index.ts

```diff
@@ -17,7 +21,7 @@ export function getFileContent(
       if (actionName !== searchedAction) {
         continue;
       }
-      if (!(typeof extractedLayers[filePath][actionName] === "string")) {
+      if (!isStringType) {
```

Comment:
I curious about how this `isStringType` works. Does it implicit pass the `actionName` const? Do you think it can raise some doubts for someone who will read the code in the future, like me? :)

Reply:
:O I don't think it did work! but probably I haven't noticed because it was fixed immediately in a later commit. let me check if any tests have failed over that (I might have not noticed due to the jest tests not running on `npm run test`, to make sure that this part is covered well, and then I'll fix this commit.

Reply:
https://github.com/snyk/snyk-docker-plugin/pull/362/commits/2e777496cb0f3126ac283599264984377c456abe

^^ this new commit fixes this issue, and adds test to lib/inputs/index.ts (which i tested and they failed prior to the change)